g3doc/user_guide/gpu.md

```diff
@@ -115,9 +115,29 @@ As a result, `nvproxy` has the following limitations:
 
 ### Supported GPUs {#gpu-models}
 
-gVisor currently supports NVIDIA GPUs: T4, L4, A100, A10G and H100. Please
+gVisor currently supports NVIDIA GPUs:
+
+*   **T4**, based on the
+    [Turing architecture](https://en.wikipedia.org/wiki/Turing_\(microarchitecture\))
+*   **A100** and **A10G**, based on the
+    [Ampere architecture](https://en.wikipedia.org/wiki/Ampere_\(microarchitecture\))
+*   **L4**, based on the
+    [Ada Lovelace architecture](https://en.wikipedia.org/wiki/Ada_Lovelace_\(microarchitecture\))
+*   **H100**, based on the
+    [Hopper architecture](https://en.wikipedia.org/wiki/Hopper_\(microarchitecture\))
+
+While not officially supported, other NVIDIA GPUs based on the same
+architectures as the above will likely work as well. This includes
+consumer-oriented GPUs such as **RTX 3090** (Ampere) and **RTX 4090** (Ada
+Lovelace).
+
+Therefore, if you encounter an incompatible workload on a GPU on one of the
+above architectures, even if on an unsupported GPU, chances are that this
+workload is also incompatible in the same manner on one of the
+officially-supported GPUs. Please
 [open a GitHub issue](https://github.com/google/gvisor/issues/new?labels=type%3A+enhancement,area%3A+gpu&template=bug_report.yml)
-if you want support for another GPU model.
+with reproduction instructions so that it can be tested against an
+officially-supported GPU.
```

Comment:
nit: this pair is hyphenated only 50% of the time in the diff. Is there a distinction b/w hyphenated and not?

Reply:
There is no distinction, the hyphenation inconsistency is accidental. Sorry, I initially missed this comment because it is on a copybara-service-generated PR. I sent a followup in https://github.com/google/gvisor/pull/10639